src/lib/Scenes/Home/index.tsx

```diff
@@ -26,13 +26,16 @@ export default class Home extends React.Component<null, null> {
   render() {
     return (
       <ScrollableTabView renderTabBar={() => <TabBar />}>
-        <Tab tabLabel="Artists">
+        {/* A thin space has been added in front of the tab label names to compensate for trailing space
```

Comment:
If you start the comment with either `TODO: ` or `FIXME: ` then these will be better to find

Reply:
https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Reply:
That's in the recommended extensions already 👍

Reply:
It is, looks very nice! `FIXME` added :)